public/locales/en/common.json

```diff
@@ -36,8 +36,9 @@
   },
   "timeout-modal": {
     "header": "Your Session Will End Soon",
-    "warning": "To protect your information, you will be logged out in {{numberOfMinutes}} minutes if you do not continue.",
-    "button": "Stay Logged In"
+    "warning": "To protect your information, you will be logged out in {{count}} minute if you do not continue",
+    "warning_plural": "To protect your information, you will be logged out in {{count}} minutes if you do not continue",
+    "button": "Return to UI Home"
```

Comment:
Updated verbiage matches [Content doc](https://docs.google.com/document/d/18ZIc1y8kq9AGxdNCVIyGKWMk5aanBz0p5SaPj2prGPc/edit?ts=60eda820#heading=h.ngy94iygfatg) - and properly handles the plural!